cmd/snap/cmd_quota.go

```diff
@@ -100,15 +114,51 @@ type cmdSetQuota struct {
 	waitMixin
 
 	MemoryMax  string `long:"memory" optional:"true"`
+	CPUMax     string `long:"cpu" optional:"true"`
+	CPUSet     string `long:"cpu-set" optional:"true"`
+	ThreadMax  string `long:"threads" optional:"true"`
```

Comment:
I think "threads" is fine and matches the field in QuotaValues, For consistency the field here should be called ThreadsMax though as well.

Reply:
I updated this now and it's threads now everywhere